pvacseq/lib/main.py

```diff
@@ -31,7 +31,7 @@ def main(args_input = sys.argv[1:]):
     parser.add_argument("output_dir",
                         help="Output directory for writing all result files"
                         )
-    parser.add_argument("-l", "--variant-peptide-length",
+    parser.add_argument("-l", "--peptide-sequence-length",
```

Comment:
It only works for --options.  the default destination is the option flag, minus any leading dashes, and with internal dashes replaced with underscores.  For positional arguments you still need to define a dest